apps/content/api/internal/assets_detail.py

```
@@ -28,27 +28,22 @@ class DetailAssetPublisherOut(Schema):
     description: str
     long_description: str
     thumbnail_url: AbsoluteUrl | None
-    publisher: DetailAssetPublisherOut = Field(alias="resource.publisher")
-    resource: DetailAssetResourceOut
+    publisher: DetailAssetPublisherOut = Field(alias="publisher")
     license: LicenseChoice
     snapshots: list[DetailAssetSnapshotOut] = Field(default_factory=list, alias="previews")
 
 
 @router.get("assets/{id}/", response=DetailAssetOut, auth=None)
 def detail_assets(request: Request, id: int):
```

Ruff (0.15.10)

[error] Function argument `id` is shadowing a Python builtin

(A002)